### PR TITLE
Implement text shadow, unify text & box shadow, improve spec complian…

### LINKED ISSUE #69: Text-shadow shorthand
We support `box-shadow`, but not `text-shadow`. RN supports everything we need to do this.

### PR DESCRIPTION
**Pull request by jacobp100**
…ce, begin new test layout

Fixes #69

### CODE REVIEW

File: src/transforms/__tests__/textShadow.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+it('textShadow omitting blur', () => {
+  expect(transformCss([['text-shadow', '10px 20px red']])).toEqual({
+    textShadowOffset: { width: 10, height: 20 },
+    textShadowRadius: 0,
+    textShadowColor: 'red',
+  })
+})
+
+it('textShadow omitting color', () => {
+  expect(transformCss([['text-shadow', '10px 20px black']])).toEqual({
```
Comment (kristerkari) [inline src/transforms/__tests__/textShadow.js]: The test description says `omitting color` but the test is the same as previous with a different color, is this correct?

File: src/transforms/__tests__/textShadow.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    textShadowOffset: { width: 10, height: 20 },
+    textShadowRadius: 0,
+    textShadowColor: 'black',
+  })
+})
+
+it('textShadow omitting blur, offset-y', () => {
+  expect(transformCss([['text-shadow', '10px 20px red']])).toEqual({
```
Comment (kristerkari) [inline src/transforms/__tests__/textShadow.js]: also this tests seems to be the same as previous ones.

File: src/transforms/__tests__/textShadow.js (hunk 1 above)
Comment (jacobp100) [reply src/transforms/__tests__/textShadow.js]: Good catch

File: src/transforms/__tests__/textShadow.js (hunk 2 above)
Comment (jacobp100) [reply src/transforms/__tests__/textShadow.js]: Another good catch, this should be removed

**Pull request merged**